### PR TITLE
Added WithSecurityContext func to pod package

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/pod/container.go [hunk 1]
```diff
@@ -111,6 +111,32 @@ func (builder *ContainerBuilder) WithSecurityCapabilities(sCapabilities []string
...
+		glog.V(100).Infof("Cannot add empty securityContext to container structure")
+
+		builder.errorMsg = "can not modify container config with empty securityContext"
+	}
+
+	if builder.definition.SecurityContext != nil {
+		glog.V(100).Infof("Cannot modify pre-existing securityContext")
+
+		builder.errorMsg = "can not modify container config with empty securityContext"
+		builder.errorMsg = "can not modify pre-existing securityContext"
```
Comment (evgenLevin) [inline pkg/pod/container.go:128]: 2 errors?

**Comment by kononovn** [comment]
@evgenLevin PTAL

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**